bundles/org.openhab.binding.pentair/README.md

```diff
@@ -72,120 +79,200 @@ The following table shows the available configuration parameters for each thing.
-| aux6                 | Switch    | Aux6 mode                                                    |
-| aux7                 | Switch    | Aux7 mode                                                    |
-| feature1             | Switch    | Feature1 mode                                                |
-| feature2             | Switch    | Feature2 mode                                                |
-| feature3             | Switch    | Feature3 mode                                                |
-| feature4             | Switch    | Feature4 mode                                                |
-| feature5             | Switch    | Feature5 mode                                                |
-| feature6             | Switch    | Feature6 mode                                                |
-| feature7             | Switch    | Feature7 mode                                                |
-| feature8             | Switch    | Feature8 mode                                                |
-| IntelliChlor         |           |                                                              |
-| saltoutput           | Number    | Current salt output % (readonly)                             |
-| salinity             | Number    | Salinity (ppm) (readonly)                                    |
-| IntelliFlo Pump      |           |                                                              |
-| run                  | Number    | Pump running (readonly)                                      |
-| drivestate           | Number    | Pump drivestate (readonly)                                   |
-| mode                 | Number    | Pump mode (readonly)                                         |
-| rpm                  | Number    | Pump RPM (readonly)                                          |
-| power                | Number    | Pump power in Watts (readonly)                               |
-| error                | Number    | Pump error (readonly)                                        |
-| ppc                  | Number    | Pump PPC? (readonly)                                         |
-
-## Full Example
-
-The following is an example of an item file (pentair.items), you can change the °F to °C if you are using metric temperature units:
+## Things & Channels
+
+### Thing: Controller
+
+Represents and interfaces with a Pentair pool controller in the system.  This binding should work for both Intellitouch and EasyTouch systems.
+Feature availability is dependent on the version of hardware and firmware versions of your specific controller.
+
+#### Synchronize Time
+
+This configuration setting will instruct the binding to automatically update the controller's clock every 24 hours with the value from the openHAB server.
+This is useful to keep the pool system clock set correct and automatically adjust for daylight savings time.
+
+| Channel Group                     | Channel           | Type               |     | Description  |
```

Comment:
This and the other tables seem a little bit distorted. Not sure what is going on, are you able to reformat these?

Reply:
hmm, they look good to me - both locally, but I also looked at the files online as part of the pull request.  Not sure what needs to change here?

Reply:
Seen another PR, you can leave it as is.